Adapter_Revit_UI/CRUD/Create.cs

```diff
@@ -82,6 +82,7 @@ protected override bool Create<T>(IEnumerable<T> objects)
             return aResult; ;
         }
 
+
```

Comment:
Was this blank line needed?

Reply:
I usually leave 2 blanks before the header.

Reply:
Ok :smile: